app/Helper/Utils.php

```diff
@@ -106,13 +107,30 @@ public static function queryCache($request,$callback,$minutes=1,$tags='default')
 
     public static function purgeCache($tag)
     {
-        if(config('cache.default') == 'redis')
-            Cache::tags([$tag,'default'])->flush();
+        if (config('cache.default') == 'redis')
+            Cache::tags([$tag, 'default'])->flush();
     }
 
-    public static function cleanStr($str): string {
+    public static function cleanStr($str): string
+    {
+
+        return str_replace(array(':', '-', '/', '*', '_', '`'), ' ', $str);
 
-        return str_replace(array(':', '-', '/', '*','_','`'), ' ', $str);
+    }
+
+    public static function getCircleUserFromRequest($request, $circle_id, $is_admin = false)
+    {
+        $profile = $request->user();
```

Comment:
is it a problem that Sanctum is assigning a profile as `$request->user()`? does that conflict with the existing uses of `$request->user` at all?